app/routes/questionnaire.py

```diff
@@ -1,23 +1,30 @@
+from __future__ import annotations
+
+from typing import Union
+
 import flask_babel
 from flask import Blueprint, g, redirect, request, url_for
 from flask_login import current_user, login_required
 from itsdangerous import BadSignature
 from structlog import get_logger
+from werkzeug import Response
```

Comment:
```suggestion
from flask import Response
```

Reply:
Using `redirect` returns `Response` from `werkzeug`. Mypy complains when using the flask class.